site/source/tutorials/vdom_abstractions.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: false
+title: vDOM Abstractions - w and v in the d module
+overview: Abstracting DOM allows us to create presentation components programatically. Dojo 2 provides a layer of abstraction between the DOM and its virtual counterpart as it exists in JavaScript and a layer of abstraction between this virtual DOM and Dojo 2 components - using a common syntax in a common module.
+---
+
+## `@dojo/widget-core/d.ts`
+
+Creating the DOM structure to support the syntax, style, and structure required to display a custom component may require a significant number of nodes. Because of this, many virtual DOM implementations have chosen single-letter function names to reduce typing and add clarity when reading code.
+
+This module, named for what would be the single-letter shorthand function name for "Dojo 2" contains two single-letter functions each named for their purpose:
+
+- `v`: is used to create an abstract version of a DOM node
+- `w`: is used to create an abstract version of a widget
+
+## Common variables
+
+A node can be thought of as an object of a certain type, customized through properties, dispatching events through callbacks, with zero or more children. Condensing this down into a function where each argument can be isolated gives us a type, a set of properties, and a list of children.
+
+Because Dojo 2 is built on TypeScript, splitting arguments into these parts allows static type enforcement that can be based on the type of node being created. It also helps explain why `v` and `w` are separate functions, as each function enforces a different set of constraints on its arguments.
```

Comment:
maybe built on TypeScript should be a coming soon link (and will eventually point to https://github.com/dojo/dojo.io/blob/master/site/source/tutorials/TypeScript_and_Dojo_2.md once that lands on the website)?